disdrodb/data_transfer/download_data.py

```diff
@@ -0,0 +1,282 @@
+import os
+import re
+import zipfile
+import pooch
+import yaml
+
+import click
+import tqdm
+
+
+# function to check if a path exists
+def check_path(path: str) -> None:
+    """Check if a path exists
+
+    Parameters
+    ----------
+    path : str
+        Path to check
+
+    Raises
+    ------
+    FileNotFoundError
+        If the path does not exist
+    """
+    if not os.path.exists(path):
+        raise FileNotFoundError(f"The path {path} does not exist. Please check the path.")
+
+
+def get_metadata_folders(folder_path: str) -> list:
+    """Return a list containing all metadata folders paths
+
+    Parameters
+    ----------
+    folder_path : str
+        folder path
+
+    Returns
+    -------
+    list
+        list of metadata folders
+    """
+
+    return [os.path.join(root, "metadata") for root, dirs, files in os.walk(folder_path) if "metadata" in dirs]
+
+
+def get_yaml_files(metadata_folders: str) -> list:
+    """Return a list containing all yaml files paths
+
+    Parameters
+    ----------
+    metadata_folders : str
+        folder path
+
+    Returns
+    -------
+    list
+        list of yaml files
+    """
+
+    return [
+        os.path.join(root, file)
+        for metadata_folder in metadata_folders
+        for root, dirs, files in os.walk(metadata_folder)
+        for file in files
+        if file.endswith(".yml")
+    ]
+
+
+def get_list_urls_local_paths(
+    raw_folder: str = None,

```

Comment:
Let's try to always ask same arguments ! 
Let's ask for `disdrodb_dir` and add "Raw" within the function !